### PR TITLE
fix: add clarifying comments, add in enable api resource

### PR DESCRIPTION
**Pull request by pattishin**
**Issue to resolve:** Small changes to clarify image replacement and general ease of applying resources after customer pubsub feedback

### CODE REVIEW

File: cloud_run_service_pubsub/main.tf [hunk 1]
```diff
@@ -33,10 +44,6 @@ resource "google_cloud_run_service_iam_binding" "binding" {
 # [END cloudrun_service_pubsub_run_invoke_permissions]
```
Comment (betsy-lichtenberg) [inline cloud_run_service_pubsub/main.tf:44]: The Google Cloud Run Service Agent service account (service-<project_number>@serverless-robot-prod.iam.gserviceaccount.com) should have the iam.serviceAccounts.getAccessToken permission via the Cloud Run Service Agent role on the project.

I fixed this by running the following gcloud command, but we should add the TF equivalent to this sample.

gcloud projects add-iam-policy-binding <project_name> --member=serviceAccount:service-<project_number>@serverless-robot-prod.iam.gserviceaccount.com --role=roles/run.serviceAgent

File: cloud_run_service_pubsub/main.tf [hunk 2]
```diff
@@ -1,18 +1,29 @@
+# Project data
```
Comment (betsy-lichtenberg) [inline cloud_run_service_pubsub/main.tf:1]: I'm getting this error:

Error: Error when reading or editing Service Account "projects/load-balancer-https/serviceAccounts/cloud-run-pubsub-invoker@load-balancer-https.iam.gserviceaccount.com": Get "https://iam.googleapis.com/v1/projects/load-balancer-https/serviceAccounts/cloud-run-pubsub-invoker@load-balancer-https.iam.gserviceaccount.com?alt=json&prettyPrint=false": dial tcp [2607:f8b0:400e:c06::5f]:443: connect: cannot assign requested address
│
│   with google_service_account.sa,
│   on main.tf line 31, in resource "google_service_account" "sa":
│   31: resource "google_service_account" "sa" {

**Review by betsy-lichtenberg — APPROVED** [review]

**Comment by comment-bot-dev** [comment]
@pattishin
Thanks for the PR! 🚀
Unfortunately it looks like some of our CI checks failed. See the [Contributing Guide](../blob/master/CONTRIBUTING.md) for details.
- ⚠️check_terraform
 Failed Terraform check. More details below. 
 ```bash 
Running terraform fmt
cloud_run_service_pubsub/main.tf
--- old/cloud_run_service_pubsub/main.tf
+++ new/cloud_run_service_pubsub/main.tf
@@ -38,7 +38,7 @@
     percent         = 100
     latest_revision = true
   }
-  depends_on                 = [google_project_service.cloudrun_api]
+  depends_on = [google_project_service.cloudrun_api]
 }
 # [END cloudrun_service_pubsub_service]
 
@@ -91,6 +91,6 @@
       x-goog-version = "v1"
     }
   }
-  depends_on                 = [google_cloud_run_service.default]
+  depends_on = [google_cloud_run_service.default]
 }
 # [END cloudrun_service_pubsub_sub]
Error: terraform fmt failed with exit code 3
Check the output for diffs and correct using terraform fmt <dir>
 ```

**Comment by comment-bot-dev** [comment]
@pattishin
Thanks for the PR! 🚀<br/>✅ Lint checks have passed.

**Comment by comment-bot-dev** [comment]
@pattishin
Thanks for the PR! 🚀<br/>✅ Lint checks have passed.

**Pull request merged**